google-cloud-firestore/lib/google/cloud/firestore.rb

```diff
@@ -21,6 +21,381 @@ module Cloud
     ##
     # # Cloud Firestore
     #
+    # Cloud Firestore is a NoSQL document database built for automatic scaling,
+    # high performance, and ease of application development. While the Cloud
+    # Firestore interface has many of the same features as traditional
+    # databases, as a NoSQL database it differs from them in the way it
+    # describes relationships between data objects.
+    #
+    # For more information about Cloud Firestore, read the [Cloud
+    # Firestore Documentation](https://cloud.google.com/firestore/docs/).
+    #
+    # The goal of google-cloud is to provide an API that is comfortable to
+    # Rubyists. Authentication is handled by {Google::Cloud#spanner}. You can
```

Comment:
Use `Google::Cloud#firestore` instead of `Google::Cloud#spanner`.

Reply:
yikes! good catch